lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -4405,24 +4405,10 @@ def test_get_student_from_request_data_user_dne(self):
         in the LMS.
         """
         with pytest.raises(ValueError) as error:
-            _get_student_from_request_data({"user": "Neo"}, self.course.id)
+            _get_student_from_request_data({"user": "Neo"})
 
         assert str(error.value) == "Neo does not exist in the LMS. Please check your spelling and retry."
 
-    def test_get_student_from_request_data_user_not_enrolled(self):
```

Comment:
Removed test as we are no longer checking for the learner's enrollment status in the `get_user()` function call.